CHANGELOG.md

```diff
@@ -11,6 +11,63 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.9.0 - 2020-08-13
+
+### Added
+
+- Add [flow exporter] feature. [Alpha - Feature Gate: `FlowExporter`]
+   * Support sending network flow records using the IPFIX protocol from each Agent ([#825](https://github.com/vmware-tanzu/antrea/pull/825) [#984](https://github.com/vmware-tanzu/antrea/pull/984), [@srikartati])
+   * Add reference cookbook to visualize exported flows using Elastic Stack ([#836](https://github.com/vmware-tanzu/antrea/pull/836), [@zyiou])
+- Support [OVS hardware offload] for Pod networking: Pods can now be assigned an SR-IOV Virtual Function. ([#786](https://github.com/vmware-tanzu/antrea/pull/786), [@moshe010])
+   * Add new CI job to validate the hardware offload functionality ([@AbdYsn])
+- Support Node MTU auto-discovery in the Antrea Agent; the user can still override this value in the Agent configuration if desired. ([#909](https://github.com/vmware-tanzu/antrea/pull/909), [@reachjainrahul])
+- Enable Antrea support for the [AKS] managed K8s service, using CNI chaining and the "networkPolicyOnly" traffic mode. ([#998](https://github.com/vmware-tanzu/antrea/pull/998), [@reachjainrahul])
+- Support for NetworkPolicy tiering (ClusterNetworkPolicy only). ([#956](https://github.com/vmware-tanzu/antrea/pull/956) [#986](https://github.com/vmware-tanzu/antrea/pull/986), [@abhiraut] [@Dyanngg])
+   * The `ClusterNetworkPolicy` Feature Gate must now be enabled for the Agent (in addition to the Controller) to activate the feature
+- Support executing Traceflow requests with antctl. ([#932](https://github.com/vmware-tanzu/antrea/pull/932), [@lzhecheng])
+- Support automatic rotation for the self-signed certificate generated by Antrea when no certificate is provided by the user. ([#1024](https://github.com/vmware-tanzu/antrea/pull/1024), [@MatthewHinton56])
+- Add new Agent Prometheus metrics for OVS flow operations. ([#866](https://github.com/vmware-tanzu/antrea/pull/866), [@yktsubo])
+- Provide a DaemonSet to automatically restart Pods on new Nodes in EKS when Antrea becomes ready: this ensures that NetworkPolicies are enforced correctly for all Pods. ([#1057](https://github.com/vmware-tanzu/antrea/pull/1057), [@reachjainrahul])
+- Add scripts to run the Antrea Agent directly without using a Pod to manage the lifecycle of the process. ([#1013](https://github.com/vmware-tanzu/antrea/pull/1013), [@ruicao93]) [Windows]
+
+### Changed
+
+- Restrict all traffic modes except for "encap" to use "Antrea Proxy" for Pod-to-Service traffic, as this greatly simplifies the datapath implementation. ([#1015](https://github.com/vmware-tanzu/antrea/pull/1015), [@suwang48404])
+- Improve Antrea Octant plugin. ([#913](https://github.com/vmware-tanzu/antrea/pull/913), [@ZhangYW18])
+   * Merge the two existing plugins (Agent / Controller Info, Tarceflow) into a single plugin / binary
```

Comment:
```suggestion
   * Merge the two existing plugins (Agent / Controller Info, Traceflow) into a single plugin / binary
```

Reply:
Thanks, will make the change after the tests complete, just before merging